WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -68,6 +86,22 @@ class PostListMainViewModel @Inject constructor(
         _postListAction.postValue(PostListAction.NewPost(site))
     }
 
+    fun updateAuthorFilterSelection(position: Int) {
+        val selection: AuthorFilterSelection = when (position) {
+            AuthorFilterSelection.ME.ordinal -> AuthorFilterSelection.ME
+            else -> AuthorFilterSelection.EVERYONE
+        }
+
+        _filterAuthorListItems.value = listOf(
+                AuthorFilterListItemUIState.Me(
+                        accountStore.account?.avatarUrl, isSelected = selection == AuthorFilterSelection.ME
+                ),
+                AuthorFilterListItemUIState.Everyone(isSelected = selection == AuthorFilterSelection.EVERYONE)
+        )
+
+        prefs.postListAuthorSelection = selection
```

Comment:
We don't need to update the `AppPrefs` when we call it from `start` method. It might be worth separating the internal `updateAuthorFilterSelection` and the one that's exposed to the activity.

Reply:
Resolved in 0da6871b3725a5f260ff79460d45623d61fa7bf3